media/js/firefox/hello/start.js

```diff
@@ -296,6 +314,10 @@
         // URL query param populated at template level in the view
         var incomingConversation = $main.data('incomingConversation');
 
+        // Get referrer and set Custom Variable. none is okay here.
+        var tourSource = $main.data('tourSource');
+        gaTrack(['_setCustomVar', 13, 'Hello FTU Referral', tourSource, 2]);
```

Comment:
Should this be set for every possible state on page load or is it meant to be associated with "Get Started" being clicked?